packages/web/src/pages/profile-page/useProfileRoute.ts

```diff
@@ -0,0 +1,21 @@
+import { cacheUsersSelectors } from '@audius/common'
+import { useSelector } from 'react-redux'
+import { useLocation } from 'react-router-dom'
+
+import { AppState } from 'store/types'
+import { getPathname } from 'utils/route'
+import { parseUserRoute } from 'utils/route/userRouteParser'
+
+const { getUser } = cacheUsersSelectors
+
+export const useProfileRoute = () => {
```

Comment:
Kinda a weird name for this, maybe `useProfileParams`?

Reply:
yeah totally agreed... yeah will do